.github/workflows/cd.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   dist:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v3
```

Comment:
This one might have issues if we use VCS versioning. If the version 0.0.0, we should add `fetch-depth: 0`.

Reply:
I have made the additions to this PR itself!